x-pack/plugins/reporting/server/browsers/chromium/driver_factory/index.js

```diff
@@ -78,13 +78,21 @@ export class HeadlessChromiumDriverFactory {
         logger: this.logger
       }));
 
+      // FIXME: not sure this event type actually exists
```

Comment:
Should exist: https://pptr.dev/#?product=Puppeteer&version=v1.10.0&show=api-event-error

Reply:
Thank you! I was looking at a completely different doc: https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#class-page

The one you link to looks better